main.go

```diff
@@ -24,9 +27,16 @@ func main() {
 
 	router := gin.Default()
 	router.GET("/albums", getAlbums)
-	router.GET("/albums/:ID", getAlbumByID)
+	//router.GET("/albums/:ID", getAlbumByID)
 
 	router.POST("/albums", postAlbums)
+
+	router.GET("/albums/:a/:b", addition)
+	router.POST("/albums/sub", substraction)
+	router.POST("/albums/multi", multipication)
+	router.POST("/albums/divid", devision)
+
+	router.POST("/albums/all", substraction, multipication, devision)
```

Comment:
Please use objective path. For ex :- if it is add you can do "/add" or "/calculate/add" and so on for div and mul and sub